unity-renderer/Assets/Scripts/MainScripts/DCL/Components/Avatar/AvatarSceneEmoteHandler.cs

```diff
@@ -51,7 +51,10 @@ public async UniTask LoadAndPlayEmote(string bodyShapeId, string emoteId)
                 //avoid playing emote if timestamp has change,
                 //meaning a new emote was trigger while this one was loading
                 if (timestamp == lamportTimestamp)
+                {
                     emotesController.PlayEmote(emoteId, lamportTimestamp);
+                    UserProfile.GetOwnUserProfile().SetAvatarExpression(emoteId, UserProfile.EmoteSource.EmoteLoop, true);
```

Comment:
Could you inject `IUserProfileBridge` so then you can do: `userProfileBridge.GetOwn()`